depends/check-ps3dev.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/sh
+#
 # check-ps3dev.sh by Naomi Peori (naomi@peori.ca)
+# Modified by luizfernandonb (luizfernando.nb@outlook.com)
 
 ## Check if $PS3DEV is set.
 if test ! $PS3DEV; then { echo "ERROR: Set \$PS3DEV before continuing."; exit 1; } fi
 
 ## Check for the $PS3DEV directory.
-( ls -ld $PS3DEV || mkdir -p $PS3DEV ) 1>/dev/null 2>&1 || { echo "ERROR: Create $PS3DEV before continuing."; exit 1; }
+( ls -ld $PS3DEV || mkdir -p $PS3DEV ) 1>/dev/null 2>&1 || { sudo mkdir $PS3DEV && sudo chown $USER:$USER $PS3DEV; } # { echo "ERROR: Create $PS3DEV before continuing."; exit 1; }
 
 ## Check for write permission.
-touch $PS3DEV/test.tmp 1>/dev/null 2>&1 || { echo "ERROR: Grant write permissions for $PS3DEV before continuing."; exit 1; }
+# touch $PS3DEV/test.tmp 1>/dev/null 2>&1 || { echo "ERROR: Grant write permissions for $PS3DEV before continuing."; exit 1; }
```

Comment:
Why was this check removed?

Reply:
If the owner of the folder is already the [user himself](https://github.com/ps3dev/ps3toolchain/pull/106/files/c435cf5a628d215c26e6137fbc5515ae41aecb03#diff-c2959f84736240516e7fa192da534108feb8b78da44178e9fadd842f9e133446R10), there is no need to check if he can write to the folder

Reply:
Even assuming that was true (it could still be write protected), you don't actually know whether or not the owner of the folder is the effective user.  The folder could exist but be owned by someone else (root for example).